### PR TITLE
AnimeDigitalNetwork Presence Fixed

### LINKED ISSUE #2682: [Bug | Presence] AnimeDigitalNetwork (ADN)
**Describe the Bug**
A user reported that when they were watching an episode the presence stayed in the "Browsing..." state.

**Steps to Reproduce**

1. Access website
2. Watch a video
3. Look at your current presence

**Expected Behavior**

It displays what video/episode you are watching instead of the "Browsing..." state.

**Screenshots**
image

**Environment**

- OS: Windows 10
- Browser: Chrome
- Version: Was unable to obtain.

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #2682
- Deleted Subtitles, cause not working and couldn't be fixed,
- Removed old DOMs & added new ones for Title.
ADN3
ADN1
ADN2

### CODE REVIEW

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 1]
```diff
@@ -11,16 +11,8 @@ presence.on("UpdateData", async () => {
...
     if (video && !isNaN(video.duration)) {
-      const title = document.querySelector(".adn-player-header a").textContent,
-        subtitle = document.querySelector(".adn-player-header span")
-          .textContent,
-        timestamps = presence.getTimestamps(
-          Math.floor(video.currentTime),
-          Math.floor(video.duration)
-        ),
-        data: PresenceData = {
+      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a"), t2 = t.textContent.toLowerCase(), title = t2.charAt(0).toUpperCase() + t2.slice(1), timestamps = presence.getTimestamps(Math.floor(video.currentTime), Math.floor(video.duration)), data = {
```
Comment (SlowLife1661) [inline websites/A/AnimeDigitalNetwork/presence.ts:14]: I would do this...
```suggestion
      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a"), 
      t2 = t.textContent.toLowerCase(), 
      title = t2.charAt(0).toUpperCase() + t2.slice(1), 
      timestamps = presence.getTimestamps(Math.floor(video.currentTime), Math.floor(video.duration)), 
      data: PresenceData = {
```

File: websites/A/AnimeDigitalNetwork/dist/metadata.json [hunk 2]
```diff
@@ -7,6 +7,10 @@
     {
       "name": "Bryan",
       "id": "293775922757238784"
+    },
+    {
+      "name": "Dark_Ville",
+      "id": "638080361179512853"
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/dist/metadata.json:13]: Not enough changes to make yourself contributor.

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 3]
```diff
@@ -11,16 +11,12 @@ presence.on("UpdateData", async () => {
...
-      const title = document.querySelector(".adn-player-header a").textContent,
-        subtitle = document.querySelector(".adn-player-header span")
-          .textContent,
-        timestamps = presence.getTimestamps(
-          Math.floor(video.currentTime),
-          Math.floor(video.duration)
-        ),
-        data: PresenceData = {
+      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a"), 
+      t2 = t.textContent.toLowerCase(),
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/presence.ts:15]: just... just do
```suggestion
      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a").textContent.toLowerCase(),
```

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 4]
```diff
@@ -35,14 +31,14 @@ presence.on("UpdateData", async () => {
...
+      if (t !== null) {
         presence.setActivity(data, !video.paused);
       }
     }
   } else if (document.location.pathname.includes("video") && !video) {
-    const title = document.querySelector("h1.sc-pzMyG.sc-jHngDS.efMrJJ")
-        .textContent,
-      data: PresenceData = {
+    const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div > div.sc-pAArZ.idjhKu > div:nth-child(2) > div > div > div > div.sc-jwDLgl.jjQkhb > div > h1"),  
+          t2 = t.textContent.toLowerCase(),
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/presence.ts:40]:
```suggestion
    const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div > div.sc-pAArZ.idjhKu > div:nth-child(2) > div > div > div > div.sc-jwDLgl.jjQkhb > div > h1").textContent.toLowerCase(),
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 5]
```diff
@@ -35,14 +30,13 @@ presence.on("UpdateData", async () => {
         delete data.endTimestamp;
       }
 
-      if (title !== null && subtitle !== null) {
+      if (t !== null) {
         presence.setActivity(data, !video.paused);
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/presence.ts:34]: yeah also you can remove this second argument since it isn't required in this situation.

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AnimeDigitalNetwork/presence.ts (hunk 5 above)
Comment (StrikerFRFX) [reply websites/A/AnimeDigitalNetwork/presence.ts:34]: will have to provide proof of working after removing the second argument as well just as a double check

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 6]
```diff
@@ -11,16 +11,11 @@ presence.on("UpdateData", async () => {
...
-      const title = document.querySelector(".adn-player-header a").textContent,
-        subtitle = document.querySelector(".adn-player-header span")
-          .textContent,
-        timestamps = presence.getTimestamps(
-          Math.floor(video.currentTime),
-          Math.floor(video.duration)
-        ),
-        data: PresenceData = {
+      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a").textContent.toLowerCase(),
+      title = t2.charAt(0).toUpperCase() + t2.slice(1),
```
Comment (Bas950) [inline websites/A/AnimeDigitalNetwork/presence.ts:15]: just wandering where you get t2 from, cause I can't find it...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/AnimeDigitalNetwork/presence.ts (hunk 6 above)
Comment (ririxidev) [reply websites/A/AnimeDigitalNetwork/presence.ts:15]: oh lmao, i forgot to change variable name when I suggested changes 😂

Comment (DarkVillager) [reply websites/A/AnimeDigitalNetwork/presence.ts]: ADN10
ADN9

File: websites/A/AnimeDigitalNetwork/presence.ts (hunk 5 above)
Comment (DarkVillager) [reply websites/A/AnimeDigitalNetwork/presence.ts]: ADN10
ADN9

**Review by ririxidev — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Pull request merged**